securedrop/source_app/utils.py

```diff
@@ -67,8 +70,8 @@ def wrapper(*args, **kwargs):
 
 
 @async
-def async_genkey(filesystem_id, codename):
-    crypto_util.genkeypair(filesystem_id, codename)
+def async_genkey(crypto_util_, filesystem_id, codename):
```

Comment:
Passing this `crypto_util_` around is a bit unclear at first blush: let's include a comment in the code for the benefit of future maintainers?

Reply:
Fixed